Pokégen.PkHex/Services/DownloaderService.cs

```diff
@@ -18,8 +18,8 @@ public DownloaderService(HttpClient httpClient)
 		
 	public async Task<PKM> DownloadPkmAsync(Uri uri, long? length, SupportedGame wantedGame)
 	{
-		long? size = null;
```

Comment:
do not rename variables names in here which do not match the project style settings